src/ui/storyTellingServerUI.js

```diff
@@ -152,7 +145,9 @@ sjrk.storyTelling.loadCustomThemeFiles = function (clientConfig) {
         href: cssUrl
     }).appendTo("head");
 
+
     $.getScript(scriptUrl, function () {
+        window.customThemeScriptLoaded = true;
```

Comment:
You should still add it to your test namespace as opposed to adding something new to the global namespace. While it's unlikely to have a collision in the test, it's still safer and more explicit. Also, I didn't mean to add it to the callback of `$.getScript` but to a real script that is passed in by your test that calls `sjrk.storyTelling.loadCustomThemeFiles`, so that you can verify that the script was indeed injected.

Reply:
I figured you meant that I should add it to the actual script file, but I was hesitant to do so since I'd be adding things to a real theme file.

The [documentation for `getScript`](https://api.jquery.com/jQuery.getScript/) indicates that the `success` callback function fires when the script has been loaded though not necessarily executed, so I can see some merit in checking with the method you're describing, but something just seems a little off. If I put something in the JS file that I check for, then I'll need to put it in every custom theme's JS file or otherwise the tests will fail for any theme that is missing that piece. Anyone who adds a new theme but not the associated flag will run into this issue.

I think this would be a perfect thing to add if/when I get mock responses for this working, in which case we can return any old arbitrary JS we like, including a flag. And in the meantime, I can properly namespace this flag or remove it altogether. What do you think?

Reply:
I didn't realize you were loading a real theme file, I thought it was loading a test one for the purpose of the tests. I'm really just suggesting that you verify that the JS file is loaded as you'd expect, and that was just one suggestion for how to approach that.